DuckDuckGo/Tab/Model/Tab.swift

```diff
@@ -1229,12 +1231,17 @@ extension Tab/*: NavigationResponder*/ { // to be moved to Tab+Navigation.swift
 
     @MainActor
     private func loadErrorHTML(_ error: WKError, header: String, forUnreachableURL url: URL, alternate: Bool) {
-        let html = ErrorPageHTMLTemplate(error: error, header: header).makeHTMLFromTemplate()
-        if alternate {
+        if error.localizedDescription == PhishingDetectionError.detected.localizedDescription {
```

Comment:
I think this would be a nice to have, but I'd like to avoid changing other logic at this point in case it breaks things. Do you think this if statement is fine for here though? I could move it to a factory that returns the HTML based on the error type instead but it'd only have two cases.

Reply:
Added the factory back in, but think moving SSL error page is probably out of scope for this.